src/features/auth/sign-in/ui/sign-in-form.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import { useState } from 'react';
+import { useTranslations } from 'next-intl';
+import { useRouter } from '@/shared/i18n/navigation';
+import { buildSignInRules } from '../model/shema';
+import { useSignIn } from '../model/use-sign-in';
+import { mapSignInError } from '@/shared/api/firebase/map-sign-in-errors';
+
+import { Button, Form, Input, Typography } from 'antd';
+
+const { Item } = Form;
+
+type FieldType = {
+  email?: string;
+  password?: string;
+};
+
+export function SignInForm() {
+  const t = useTranslations('SignInForm');
+  const [form] = Form.useForm<FieldType>();
+  const rules = buildSignInRules(t);
+  const router = useRouter();
+
+  const { mutate, loading } = useSignIn();
+  const [apiError, setApiError] = useState<string | null>(null);
+
+  const onFinish = async (values: FieldType) => {
+    try {
+      setApiError(null);
+      await mutate({
+        email: values.email!,
+        password: values.password!,
+      });
+
+      form.resetFields(['email', 'password']);
+      router.push('/');
+    } catch (e) {
+      const { field, key } = mapSignInError(e);
+      const msg = t(`apiErrors.${key}`);
+
+      if (field) {
+        form.setFields([{ name: field, errors: [msg] }]);
+      } else {
+        setApiError(msg);
+      }
+    }
+  };
+
+  return (
+    <Form form={form} layout="vertical" autoComplete="off" onFinish={onFinish}>
+      <Item>
+        <Typography.Text type="danger">{apiError || ' '}</Typography.Text>
```

Comment:
Add `const { Text } = Typography` at the top of the file.

```suggestion
        <Text type="danger">{apiError || ' '}</Text>
```